ingestion_tools/scripts/db_import.py

```diff
@@ -27,6 +27,7 @@ def cli():
 
 def db_import_options(func):
     options = []
+    options.append(click.option("--import-alignments", is_flag=True, default=False))
```

Comment:
Curious on why we are adding alignment import support to v1 db import?

Reply:
Updated PR description with the reasoning!